src/ViewModels/InitializationViewModel.cs

```diff
@@ -13,9 +13,17 @@
 
 namespace DevHome.ViewModels;
 public class InitializationViewModel : ObservableObject
-{
+{
+#if CANARY_BUILD
+    private const string GitHubExtensionStorePackageId = "9N806ZKPW85R";
+    private const string GitHubExtensionPackageFamilyName = "Microsoft.Windows.DevHomeGitHubExtension.Canary_8wekyb3d8bbwe";
+#elif STABLE_BUILD
     private const string GitHubExtensionStorePackageId = "9NZCC27PR6N6";
     private const string GitHubExtensionPackageFamilyName = "Microsoft.Windows.DevHomeGitHubExtension_8wekyb3d8bbwe";
+#else
+    private const string GitHubExtensionStorePackageId = "";
```

Comment:
What happens in StartAppInstallAsync() when this is empty? Maybe a comment would be helpful.

Reply:
Honestly, nothing really happens.  The API fires an exception which we catch and log, but to the user the initialization page just finishes immediately and Dev Home works as expected.  Still... added a null check to be a bit cleaner :)